src/utils/validations.js

```diff
@@ -123,6 +123,11 @@ export const validateVmLikeEntityName = (value, vmSettings, props) => {
       );
 };
 
+export const validateCloudInitHostName = value => {
+  const dnsValidation = validateDNS1123SubdomainValue(value);
+  return dnsValidation && dnsValidation.type !== EMPTY_ERROR ? dnsValidation : null;
```

Comment:
a bit differently

```suggestion
  return dnsValidation && dnsValidation.isEmptyError ? null : dnsValidation;
```

Reply:
That looks much cleaner, thanks. =) I've updated the commit.